doc/user/content/sql/recursive-ctes.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Recursive CTEs
+description: "Recursive CTEs are used to define recursive queries."
+menu:
+  main:
+    parent: 'transform'
+    name: Recursive CTEs
+    weight: 23
+---
+
+Recursive CTEs can be used to define computations on recursively defined structures (such as trees or graphs) implied by the data in your tables.
+
+## Syntax
+
+### with_recursive_cte
+
+{{< diagram "with-recursive-ctes.svg" >}}
+
+### recursive_cte_binding
+
+{{< diagram "recursive-cte-binding.svg" >}}
+
+Field | Use
+------|-----
+**RETURN AT RECURSION LIMIT $n** | An optional clause indicating that the fixpoint computation should stop after `$n` iterations and use the current values computed for each `recursive_cte_binding` in the `select_stmt`. This could be useful when debugging and validating the correctness of recursive queries, or when when you know exactly how many iterations you want to have, regardless of reaching a fixpoint. See the [Examples](#examples) section for an example.
```

Comment:
```suggestion
**RETURN AT RECURSION LIMIT $n** | An optional clause indicating that the fixpoint computation should stop after `$n` iterations and use the current values computed for each `recursive_cte_binding` in the `select_stmt`. This could be useful when debugging and validating the correctness of recursive queries, or when you know exactly how many iterations you want to have, regardless of reaching a fixpoint. See the [Examples](#examples) section for an example.
```